### ISSUE TITLE
getActiveMatches throws a "not a function" error

### ISSUE THREAD

**Issue by bkoelper** [LionelBergen/MundoScript#2]
When I make a basic call `getActiveMatches(accountInfo)` I get the following returned back to me:

```
(node:8488) UnhandledPromiseRejectionWarning: TypeError: fullPerks.getByPerkIds is not a function
    at replaceObjectFoundByKeyRunes (/node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:467:25)
    at replaceObjectFoundByKeyRunes (/node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:462:7)
    at replaceObjectFoundByKeyRunes (/node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:462:7)
    at replaceObjectFoundByKeyRunes (/node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:462:7)
    at replaceObjectKeysWithValues (/node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:438:2)
    at loadObject (/node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:242:3)
    at /node_modules/leagueapiwrapper/LeagueAPI/LeagueAPI.js:231:4
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

My code that's calling this is a very, **very** basic call, because I am still familiarizing myself with what data I can get.

```javascript
leagueAPI.getActiveGames(accountInfo).then(data => {
    console.log('data =', data);
});
```

I have confirmed that my `accountInfo` parameter is populated from a call right after initialization of the API. The `getActiveMatches` call is made much later, so that shouldn't be a problem.

If you'd like anything else, please let me know. Or let me know what I'm doing wrong, because it could always be something silly like that.

**Comment by LionelBergen** [comment]
Thanks for the detailed report.
I will take a look at this tomorrow evening if not sooner.

**Comment by LionelBergen** [comment]
@bkoelper Sorry for delay on this and thanks again for pointing the issue out. It's fixed now, with the latest release (2.0.8).
